tests/acceptance/features/bootstrap/PublicWebDavContext.php

```diff
@@ -548,25 +548,28 @@ public function thePublicHasUploadedFileWithPasswordAndContentUsingPublicWebDAVA
 	/**
 	 * @param string $filename target file name
 	 * @param string $body content to upload
+	 * @param string $publicWebDAVAPIVersion
 	 *
 	 * @return void
 	 */
-	public function publiclyOverwritingContent($filename, $body = 'test') {
-		$this->publicUploadContent($filename, '', $body, false);
+	public function publiclyOverwritingContent($filename, $body = 'test', $publicWebDAVAPIVersion='old') {
+		$this->publicUploadContent($filename, '', $body, false, [], $publicWebDAVAPIVersion);
 	}
 
 	/**
-	 * @When the public overwrites file :filename with content :body using the old WebDAV API
+	 * @When /^the public overwrites file "([^"]*)" with content "([^"]*)" using the (old|new) WebDAV API$/
```

Comment:
you can use it like this. regex can be omitted
```suggestion
	 * @When the public overwrites file :filename with content :body using the :type WebDAV API
```